### DISCUSSION TITLE
workflow_dispatch has NO button

### DISCUSSION THREAD

**Discussion by flagrant99** [community/community#66146] [Actions] [answered]
### Select Topic Area

Question

### Body

I added 

on:
  workflow_dispatch:

There is no button. Where is this fabled button supposed to be located?

**Comment by flagrant99** [comment]
So I just read that the workflow must be in the default branch? Why? I have to work and test in master? Why? Normally we do work in a branch and when done and tested we push to master.

**Reply by jemshit** [reply, marked as answer]
Yeah it has to be on main or sth. One of the stupid features of Actions
